llama_index/llms/base.py

```diff
@@ -76,17 +76,38 @@ def __str__(self) -> str:
 
 
 class LLMMetadata(BaseModel):
-    """LLM metadata."""
-
-    context_window: int = DEFAULT_CONTEXT_WINDOW
-    num_output: int = DEFAULT_NUM_OUTPUTS
-    is_chat_model: bool = False
-    is_function_calling_model: bool = False
-    # By default we don't know the model name. We can set it automatically for
-    # some types, but custom predictors (like locally loaded models) we won't
-    # know.
-    # Used for tests, logging, and sanity checks
-    model_name: str = "unknown"
+    context_window: int = Field(
+        default=DEFAULT_CONTEXT_WINDOW,
+        description="Number of tokens the model can intake when generating a response.",
+    )
+    num_output: int = Field(
+        default=DEFAULT_NUM_OUTPUTS,
+        description="Number of tokens the model can output when generating a response.",
+    )
+    is_chat_model: bool = Field(
+        default=False,
+        description=(
+            "Set True if the model supports chatting, like OpenAI's"
+            " /chat/completions endpoint."
+        ),
+    )
+    is_function_calling_model: bool = Field(
```

Comment:
Can you guys lmk what you think of this `description`? It's the only one I am not quite sure of

Reply:
"The llm supports a function calling messages, similar to OpenAI's function calling API

Reply:
Got it, thanks! Implemented